CHANGELOG.md

```diff
@@ -12,6 +12,15 @@ Subheadings to categorize changes are `added, changed, deprecated, removed, fixe
 
 This release has an [MSRV][] of 1.75.
 
+## [0.4.0]  - 2024-11-11
```

Comment:
Why is this release date here? This hasn't been released.

Reply:
Oh, I thought I did.